hwbench/environment/hardware.py

```diff
@@ -45,6 +46,7 @@ def __init__(self, out_dir: pathlib.Path, monitoring_config):
         self.vendor = first_matching_vendor(out_dir, self.dmi, monitoring_config)
         self.vendor.save_bios_config()
         self.vendor.save_bmc_config()
+        self.block = Block_Devices(out_dir)
```

Comment:
This must be mocked, I'm sure this is breaking the tests.

Reply:
Yes. But FTR this does not break the tests.